src/view/pages/Farmer/tabs/Buy/index.tsx

```diff
@@ -58,6 +65,7 @@ class Buy extends React.Component<PropsType, OwnStateType> {
           <DataTable
             headers={['Date', 'Product', 'Quantity', 'Price']}
             values={this.props.purchaseTransactions}
+            onEntryPress={this.onEntryPress}

```

Comment:
added all this stuff to stop it complaining. It will need to be put in place later anyway